packages/react-app/src/components/pages/Bounties/Form/ExpansionField.tsx

```diff
@@ -0,0 +1,47 @@
+import { Accordion, AccordionButton, AccordionItem, AccordionPanel } from '@chakra-ui/accordion';
+import { FormLabel } from '@chakra-ui/form-control';
+import { Box } from '@chakra-ui/layout';
+
+const ExpansionField = ({
+	label, children, value, onChange,
+}: {
+  label: string,
+  children: React.ReactNode,
+  value: boolean,
+  onChange: (v: boolean) => void
+}): JSX.Element => {
+	return (
+		<Accordion
+			w="full"
+			allowToggle
+		>
+			<AccordionItem
+			>
+				<AccordionButton
+					onClick={() => {
+						onChange(value = !value);
```

Comment:
why is `onChange` a param?

Reply:
Return the onchange function but detect the toggle